test/DEFAULT_FASTRTPS_PROFILES.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dds xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+  <!-- INTRAPROCESS -->
+  <library_settings>
+    <intraprocess_delivery>FULL</intraprocess_delivery>
+  </library_settings>
+  <profiles>
+     <publisher profile_name="publisher profile" is_default_profile="true">
+         <qos>
+             <publishMode>
+                 <kind>SYNCHRONOUS</kind>
+             </publishMode>
+         </qos>
+         <historyMemoryPolicy>PREALLOCATED_WITH_REALLOC</historyMemoryPolicy>
+     </publisher>
+
+     <subscriber profile_name="subscriber profile" is_default_profile="true">
+         <historyMemoryPolicy>PREALLOCATED_WITH_REALLOC</historyMemoryPolicy>
+     </subscriber>
```

Comment:
The formatting regressed here.

Reply:
https://github.com/ros2/buildfarm_perf_tests/pull/26/commits/ffe6e7bdbefceea74a77750cddd54b54e346ad63